TrustKit/Pinning/ssl_pin_verifier.m

```diff
@@ -10,16 +10,19 @@
  */
 
 #import "ssl_pin_verifier.h"
+#import "TSKSPKIHashCache.h"
 #import "../Dependencies/domain_registry/domain_registry.h"
-#import "public_key_utils.h"
-#import "../TrustKit+Private.h"
 #import "../configuration_utils.h"
+#import "../TSKLog.h"
 
 
 #pragma mark SSL Pin Verifier
 
-TSKPinValidationResult verifyPublicKeyPin(SecTrustRef serverTrust, NSString *serverHostname, NSArray<NSNumber *> *supportedAlgorithms, NSSet<NSData *> *knownPins)
+TSKPinValidationResult verifyPublicKeyPin(SecTrustRef serverTrust, NSString *serverHostname, NSArray<NSNumber *> *supportedAlgorithms, NSSet<NSData *> *knownPins, TSKSPKIHashCache *hashCache)
 {
+    NSCParameterAssert(serverTrust);
+    NSCParameterAssert(supportedAlgorithms);
+    NSCParameterAssert(knownPins);
     if ((serverTrust == NULL) || (supportedAlgorithms == nil) || (knownPins == nil))
```

Comment:
Asserts are only enabled for debug builds. In release, we still need to gracefully handle the cases.